hw0/README.rst

```diff
@@ -44,6 +44,6 @@ run the following commands:
 .. code-block:: bash
 
   cd nsdhw_21au/hw0/username
-  ../validate.py | grep "GET POINT"
+  ../validate.sh | grep "GET POINT"
```

Comment:
You should not include change from another branch.

Reply:
I am not sure how this happened, may I ask what led to this and how I can resolve this issue?

Reply:
You should branch from `hw1`.  I guess you branched out from another branch.